ansible/Makefile

```diff
@@ -43,7 +43,7 @@ ifndef LOG_SINCE
 endif
 	ansible-playbook ${ANSIBLE_DIR}/get-logs.yml \
 		-i ${ENV_DIR}/gen/terraform-inventory.yml \
-		-i ${ENV_DIR}/inventory.yml \
```

Comment:
Just checking. is the dropping of `.yml` intentsional herE?

Reply:
the whole change was not meant to be in here. Will remove it.